package-lock.json

```diff
@@ -9551,9 +9551,15 @@
             }
         },
         "node_modules/nanoid": {
-            "version": "3.3.4",
-            "resolved": "https://registry.npmjs.org/nanoid/-/nanoid-3.3.4.tgz",
-            "integrity": "sha512-MqBkQh/OHTS2egovRtLk45wEyNXwF+cokD+1YPf9u5VfJiRdAiRwB2froX5Co9Rh20xs4siNPm8naNotSD6RBw==",
+            "version": "3.3.6",
+            "resolved": "https://registry.npmjs.org/nanoid/-/nanoid-3.3.6.tgz",
+            "integrity": "sha512-BGcqMMJuToF7i1rt+2PWSNVnWIkGCU78jBG3RxO/bZlnZPK2Cmi2QaffxGO/2RvWi9sL+FAiRiXMgsyxQ1DIDA==",
+            "funding": [
```

Comment:
@rcooney, Do you know anything about what this added section for 'funding' is for? The Dependabot automatic update added it. Do we need to have it included or should we remove it?

Reply:
That isn't anything to worry about. It is a built-in aspect of npm packages. Packages are able to declare that they are looking for funding. This happens to end up in the generated `package-lock.json` but it is normal, and not anything that is visible to end users of what we build.

Reply:
Great. Thanks for the info.